ui/app/pages/first-time-flow/metametrics-opt-in/index.scss

```diff
@@ -25,12 +25,10 @@
   }
 
   &__title {
+    @include H1;
```

Comment:
Results in a small change from 42px to 40px

Reply:
grabbed an after, i'll get a before soon:
![image](https://user-images.githubusercontent.com/4448075/88585803-21ea8980-d019-11ea-97de-fa5d4b2a6787.png)

Reply:
Seems fine to me!